vncviewer/EmulateMB.h

```diff
@@ -21,29 +21,50 @@
 
 #include <rfb/Timer.h>
 #include <rfb/Rect.h>
+#include <rdr/types.h>
 
 class EmulateMB : public rfb::Timer::Callback {
 public:
-  EmulateMB();
+  EmulateMB(rdr::U32 emulateMBModKey);
 
   void filterPointerEvent(const rfb::Point& pos, int buttonMask);
 
+  bool filterKeyPress(int keyCode, rdr::U32 keySym);
+  bool filterKeyRelease(int keyCode);
```

Comment:
Let's have the same symmetry with `filterPointerEvent()` in that these call things on their own, and hence have no return value.